src/app/create-article/create-article.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { UpdateDataService } from '../update-data.service';
+
+@Component({
+  selector: 'app-create-article',
+  templateUrl: './create-article.component.html',
+  styleUrls: ['./create-article.component.css']
+})
+
+export class CreateArticleComponent implements OnInit {
+  constructor(private channelService: UpdateDataService) { 
+    this.channelService.updateFlag(0);
+  }
+
+  ngOnInit() { }
+
+  saveArticle(heading:String,shortDescription:String,content:String,date:String,author:String, url:String){
+    alert("Heading : "+heading+"\nShort Description : "+shortDescription+" \nContent"+content+"\nDate : "+date+"\nAuthor : "+author+"\nShort Url : "+url);
+    console.log("Heading : "+heading+"\nShort Description : "+shortDescription+" \nContent"+content+"\nDate : "+date+"\nAuthor : "+author+"\nShort Url : "+url);
```

Comment:
Please remove `alerts` and `console.log` statements before you commit the changes

Reply:
It was a part of HomeTask 
We had console.log the details that we get from the above input fields.